ee/rbac/lib/rbac/okta/saml/jit_provisioner/add_user.ex

```diff
@@ -116,7 +116,9 @@ defmodule Rbac.Okta.Saml.JitProvisioner.AddUser do
       :ok
     else
       {:ok, rbi} = RoleBindingIdentification.new(user_id: user_id, org_id: org_id)
-      {:ok, nil} = RoleManagement.assign_role(rbi, role_id, :saml_jit)
+      # Although this role is assigned through :saml_jit, it can be updated and
```

Comment:
```suggestion
      # Although this role is assigned through saml_jit, it can be updated and
```

Maybe we could remove this as atom reference since we are not going to have it anymore